test/fixtures/statement/throw/migrated_0003.js

```diff
@@ -0,0 +1,2 @@
+throw `Error:
+Generic Error`;
```

Comment:
Since this is about template literal, the more proper place for this test is in `test/fixtures/ES6/template-literals/`. Also, use a more descriptive name.